providers/rubygems/rubygems.go

```diff
@@ -0,0 +1,21 @@
+package rubygems
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("rubygems", &RubyGemsProvider{})
+}
+
+// RubyGemsProvider adheres to the Provider interface.
+type RubyGemsProvider struct {
+}
+
+// BuildURI generates a search URL for RubyGems.
+func (p *RubyGemsProvider) BuildURI(q string) string {
+	return fmt.Sprintf("https://rubygems.org/search?utf8=%%E2%%9C%%93&query=%s", url.QueryEscape(q))
```

Comment:
They both work. The `utf8=%%E2%%9C%%93` one was made by copy and pasting into `vi`. `utf8=✓` is prettier.